google/resource_compute_security_policy.go

```diff
@@ -53,6 +53,14 @@ func resourceComputeSecurityPolicy() *schema.Resource {
 				Description: `The project in which the resource belongs. If it is not provided, the provider project is used.`,
 			},
 
+			"type": {
```

Comment:
Given this is an enum, a validateFunc for enum may be the best option.

Reply:
See this for an example: https://github.com/hashicorp/terraform-provider-google/blob/9eef5a485ec52fa52fcfd11e2902dd062c7a0b37/google/resource_cloudbuild_trigger.go#L417-L422

Reply:
Added validation in latest commit